manifest.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright 2024 [TODO: COPYRIGHT HOLDER NAME]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// manifest.js
+
+let rulesetFilePaths = [];
+
+// Uploading and displaying manifest file
+const manifestFileInput = document.getElementById('manifestFileInput');
+manifestFileInput.addEventListener('change', (event) => {
+    const manifestFile = event.target.files[0];
+    if(manifestFile){
+        const reader = new FileReader();
+        reader.onload = function(e) {
+            try {
+                const manifestObject = JSON.parse(e.target.result);
+                let manifestSyntaxError = isValidManifest(manifestObject);
```

Comment:
`manifestSyntaxError` feels like an inverted name, since if it's true there **isn't** a syntax error. Should we find a better name for this variable perhaps?